packages/nuxt/src/runtime/plugin.ts

```diff
@@ -42,18 +42,19 @@ export default defineNuxtPlugin((nuxtApp) => {
   const head = getGlobalProperty(app, '$head')
 
   if (head) {
-    const colorConfig = getGlobalProperty(app, '$vaColorConfig')
-
-    if (colorConfig) {
-      // TODO: Remove. This is a temporary fix for the issue with vuestic-ui.
-      const renderCSSVariables = colorConfig.renderCSSVariables || colorConfig.renderCSSVarialbes
-
-      // Add reactive CSS variables to head so they are taken from colorConfig
-      head.addHeadObjs(ref({
-        htmlAttrs: {
-          style: renderCSSVariables()
-        }
-      }))
-    }
+    watchEffect(() => {
+      const colorConfig = getGlobalProperty(app, '$vaColorConfig')
+
+      if (colorConfig) {
+        const renderCSSVariables = colorConfig.renderCSSVariables
```

Comment:
May be will be better to handle & re-render only changed-new colors from color config? In performance purpose.

Reply:
I think it is a minor thing